libs/core-scanner/src/pages/robots-txt.ts

```diff
@@ -72,7 +72,7 @@ const findRobotsCrawlDelay = (
   robotsTxt: string,
 ) => {
   const directives = robotsTxt.split('\n');
-  let crawlDelay: number;
+  let crawlDelay = -1;
```

Comment:
Can we initialize this to NULL?

```suggestion
  let crawlDelay = null;
```